### ISSUE TITLE
Needs some example Songs

### ISSUE THREAD

**Issue by sttng** [stoneface86/trackerboy#13]
I think your tracker should come with some example songs and demos for users to play around with.
Feel free to add this one: https://github.com/sttng/gb-stuff/blob/main/Konami_Logo.tbm

**Comment by stoneface86** [comment]
Agreed, I've been meaning to do this at some point. I'll ask around to see if anyone wants to have their modules packaged as an example. Will definitely add the one you linked as well!

Releases are currently packaged as a zip/tarball with the executable, license and change log, so I'll change it to include an `examples/` sub directory with example modules.

For anyone else reading this issue, feel free to link your modules if you want them included! Just make sure you fill out the artist information in the module properties dialog (Ctrl+P), and leave any additional comments as well (Module | Comments...)

**Comment by stoneface86** [comment]
Added some [here](../tree/develop/examples/)!

All files in this folder will be included in future releases from now on.
